css/modal.less

```diff
@@ -2,6 +2,7 @@
 
     .modal-backdrop {
         opacity: 0.3;
+        z-index: 2;
```

Comment:
Do you remember what this was trying to get in front of?
Just wondering as I can't see many z-indexes about

Reply:
The search input box was displaying above the backdrop